recipes-bsp/xt-rpi-u-boot-scr/files/boot.cmd.dtapply

```diff
@@ -0,0 +1,3 @@
+fatload @@BOOT_MEDIA@@ 0 @@XEN_DTBO_ADDR@@ @@DTBO@@
```

Comment:
Here in the commit description you don't have "." at the end, but I can see "." on the next commit description. please fix commit format